README.md

```diff
@@ -80,6 +80,37 @@ Create an `examples` object on each parameter that needs to go into a group in t
+| Missing required parameters | No example is provided for a parameter marked as required | Add examples for all required parameters or remove the required flag if the parameter is not required |
+
+## Response Validation Failures
+If one of these validation failures occur the rest of the response will not be validated.
+
+| Failure               | Description                                                        | Fix                          |
+| --------------------- | ------------------------------------------------------------------ | ---------------------------- |
+| Status code mismatch  | Actual response has a status code that is not included in the OAS  | Add the missing status code  |
+| Content type mismatch | Actual response has a content type that is not included in the OAS | Add the missing content type |
+
+## Schema Validation Failures
+These failures can occur for parameters and responses.
+
+| Failure                   | Description                                                                | Fix                                                                                           |
+| ------------------------- | -------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- |
+| Unexpected null value     | Actual object is null, but the schema does not allow null values           | If null values should be allowed for this object set the nullable field to true in the schema |
+| Type mismatch             | Actual object type does not match type defined in the schema               | Update the schema so type is set correctly                                                    |
+| Duplicate enum values     | Schema enum contains duplicate values                                      | Remove duplicate values from the enum                                                         |
+| Enum mismatch             | Actual object does not match a value in the schema enum                    | Update the enum to contain all valid values                                                   |
+| Missing items field       | Schemas for arrays must include the items field                            | Set the items field on array schemas                                                          |
+| Missing properties field  | Schemas for objects must include the properties field                      | Set the properties field on object schemas                                                    |
+| Properties mismatch       | Actual object contains properties not present in the schema                | Update the schema so all valid properties are included                                        |
+| Missing required property | Actual object does not contain a property marked as required in the schema | Update the schema so that only required properties are marked as required                     |
```

Comment:
❓ _question_: Is the issue always going to be the schema? Or is the answer sometimes to change the actual object? All of the fixes we've listed are about changing the schema rather than the object.

Reply:
This is a good question... I think for the most part with these it's likely that the API changed and the documentation has fallen behind or maybe there was something wrong with the documentation to start with. But it is possible that to fix some of these changes could be made to the APIs themselves. Do others have thoughts on this? @RockslideJr